.github/workflows/end2end-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: e2e tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+env:
+  PACKAGEPATH: ${{ github.workspace }}/go/src/github.com/${{ github.repository }}
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.PACKAGEPATH }}
+      - name: Build image
+        working-directory: ${{ env.PACKAGEPATH }}
+        run: docker build . --file Dockerfile --tag xdsrelay
```

Comment:
Good question. The documentation doesn't mention anything, I'll see what I can find out.

Reply:
This is not documented anywhere, but I tested this on a private repo of my own and the generated docker image does not count against my storage quota. I don't have permissions to check the usage of the `xds-relay` project, but I'm going to assume that it's the same (since we're not really generating any egress calls to an outside registry or anything).

Reply:
sounds good. We can cleanup if needed later.